packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -356,10 +356,10 @@ export interface BasicCrawlerOptions<Context extends CrawlingContext = BasicCraw
  */
 export interface CrawlerExperiments {
     /**
-     * Enables the use of the new RequestQueue API, which allows multiple clients to use the same queue,
-     * by locking the requests they are processing for a period of time.
+     * Switches the RequestQueue to use the old, non-locking API. This is a temporary option to allow fallbacks if the new
+     * default causes issues. Please open an issue if you encounter problems with the new API.
      */
-    requestLocking?: boolean;
+    disableRequestLocking?: boolean;
```

Comment:
this change is technically breaking, isnt it? i would keep the same flag and just change the default value maybe?

Reply:
We've documented this interface as not respecting semver conventions so we can add/remove flags as we seem fit. We can keep it in I suppose and mark it as deprecated / won't do anything

Reply:
I don't like this approach. We introduces an experiment, and now we break it for users who tried it? Makes no sense to me this way.

Reply:
Thats, by definition, what an experiment is. Something that comes, may stick, but can vanish at any time 👀. It's the same as trying UI experiments on web where only x% gets this change, which may or may not stick.

That said, to avoid TS compile errors, I'll bring the field back and mark it as deprecated

Reply:
I just dont like the idea of breaking builds for no reason - people opted in to something that will be the new default, why should we break anything for those? Feels a bit rude to me, they helped us test the feature, we should be nicer :P

Reply:
I agree. What about keeping this flag and deprecating it and removing it in the following version?

I would do the another flag to use old request queue if necessary, it should not be necessary in experimental features, maybe in root of crawler options.

Reply:
> I would do the another flag to use old request queue if necessary, it should not be necessary in experimental features, maybe in root of crawler options.

Flag is added in experimental to stay in the same place. Realistically, I blundered calling this option `experiments` and not `featureFlags` (something to fix in v4 maybe), but I don't think it matters too much